packages/framework-core/src/booster-register-handler.ts

```diff
@@ -10,6 +10,9 @@ import {
 
 export class RegisterHandler {
   public static async handle(config: BoosterConfig, logger: Logger, register: Register): Promise<void> {
+    if (register.eventList.length == 0) {
+      return
+    }
```

Comment:
Without this check, the provider throws an error when we try to store an empty array of events